iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpWithPassword.swift

```diff
@@ -13,45 +13,46 @@ import TabBar
 import shared
 
 struct SignUpWithPassword: View {
-	
-	@ObservedObject private var viewModel: SignUpWithPasswordViewModel
-	
-	public init(viewModel: SignUpWithPasswordViewModel) {
-		self.viewModel = viewModel
-	}
-	
-	var body: some View {
-		VStack(alignment: .leading) {
-			SignUpInfoView(title: viewModel.signUpWithPasswordTitle, description: viewModel.signUpWithPasswordInfo)
-			Spacer().frame(maxHeight: .infinity)
-			
-			GeometryReader { geometry in
-				
-				let viewModel = SignUpUserEntryViewModel(
-					isSignupWithEmail: false,
-					textFieldPlaceholder: self.viewModel.signUpWithPasswordPlaceHolder,
```

Comment:
The only changes in this file were that I renamed the strings in the viewModel to be `signUpWhatever` instead of `SignUpWithPasswordWhatever`